### DISCUSSION TITLE
max file upload size per front-end increase

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3886] [Q&A]
hello everyone, I was trying to upload a file using the add dataset option, but when trying to upload a file it says that the maximum size is 2mb, I was investigating and modifying the nginx, also adding a line in the settings.php of drupal, but it does not work, I ask you a great favor if you can tell me how to make the change or where the maximum file upload size should be changed, nginx shows me the 413 error. Thank you very much

**Comment by jastraat** [comment]
You may need to increase the PHP settings post_max_size and upload_max_filesize. These are outside of Drupal. Depending on your host, changing these two PHP settings can be done in a number of places with the most likely being php.ini or .htaccess (depending on your hosting situation).

To confirm if that is the problem, go to the status report page ( /admin/reports/status ), click 'more information' next to your PHP version, and search for the two settings above to check what the current limits are.

**Comment by tridoxx** [comment]
@jastraat 
Hello, thanks for your answer, if you already tried that, as they are dockers containers, if I can't find the ini.php file, it can't be found, check the report file and the php says that the maximum is 2mb, I don't know if Maybe someone had already made the change or if it should be passed as an environment variable, but I can't find how to make that change, thank you very much for your answer anyway .

**Comment by janette** [comment]
@tridoxx for questions not related to DKAN you may have better luck on a docker forum or contacting your hosting service
https://forums.docker.com/t/how-to-get-access-to-php-ini-file/68986/21
